lib/client.js

```diff
@@ -41,10 +41,8 @@ var SCROLLBACK_DELAY_MS = 3000;
 var CRYPTO_ENABLED = false;
 
 try {
-    var Olm = require("olm");
-    if (Olm.Account && Olm.Session) {
-        CRYPTO_ENABLED = true;
-    }
+    var OlmDevice = require("./OlmDevice");
+    CRYPTO_ENABLED = true;
```

Comment:
Does not checking for the properties mean it will fail if you stub out the olm library? (eg. telling webpack to replace it with a stub)